src/dscanner/analysis/run.d

```diff
@@ -354,6 +368,38 @@ bool shouldRun(check : BaseAnalyzer)(string moduleName, const ref StaticAnalysis
 	return true;
 }
 
+bool shouldRunDmd(check : BaseAnalyzerDmd!ASTBase)(string moduleName, const ref StaticAnalysisConfig config)
```

Comment:
Whenever you add a public function, please document it. I have no idea what this function is supposed to be doing.